app/models/scheme.rb

```diff
@@ -5,6 +5,7 @@ class Scheme < OpenStruct
   extend GdsApi::Helpers
 
   FACET_KEYS = [:areas, :business_sizes, :locations, :sectors, :stages, :support_types]
+  WHITELISTED_AREA_CODES = ["LBO","CTY", "DIS", "UTA", "EUR", "COI", "GLA", "MTD", "WMC", "LGD", "SPE", "WAE", "LGE", "EUP"]
```

Comment:
Nothing breaks (apart from the test testing that "EUP" results are whitelisted), and having consistent lists would seem sensible... :confused: